osu.Game/Skinning/Components/BeatmapAttributeText.cs

```diff
@@ -78,6 +87,7 @@ protected override void LoadComplete()
             beatmap.BindValueChanged(b =>
             {
                 updateBeatmapContent(b.NewValue);
+                getMaxPP();
```

Comment:
I will not allow this. This should be only calculated *if needed*. If I allow this, someone else is going to come along and tack on another half-baked async flow that runs here even if the value is *not actually displayed*.

Reply:
maybe we need to re consider the work flow of this component?
The data of this module is calculated in advance.

Or hard-code this for the time being?